pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -43,7 +43,7 @@ import (
 	"github.com/mitchellh/mapstructure"
 	gcfg "gopkg.in/gcfg.v1"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Whats the reason for this change?

Reply:
Good point!

Reply:
Apparently this a new convention. The linter is doing that automatically.

Reply:
IMHO it would make more sense to name it corev1 then.

Reply:
Alright, using now `corev1`.